server/src/main/java/org/opensearch/gateway/remote/ClusterStateDiffManifest.java

```diff
@@ -184,6 +190,7 @@ public ClusterStateDiffManifest(StreamInput in) throws IOException {
         this.hashesOfConsistentSettingsUpdated = in.readBoolean();
         this.clusterStateCustomUpdated = in.readStringList();
         this.clusterStateCustomDeleted = in.readStringList();
+        this.indicesRoutingDiffPath = in.readString();
```

Comment:
Will this fail if diffPath doesnt exist in manifest?

Reply:
Made optional read.